### ISSUE TITLE
[4.0] [a11y] and consistency in status tooltips

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#16301]
From an accessibility and usability perspective the tooltips for the published state and featured icons should be consistent and as they are also actions then it should be clear what would happen when you click on them

### Steps to reproduce the issue
this is true in J4 and J3

Create 5 content items
1. Published
2. Published and start publishing in the future
3. Published and end publishing in the past
4. Unpublished
5. Published and featured

If you check the featured icon then it only says
`Toggle featured status 
`
It should also indicate the current status as well 

If you check the published icon tooltips you will see that there is no consistency in the information they convey
1. Published and is Current.<br>Start: 2017-05-26 09:53:20
2. Published, but is Pending.<br>Start: 2017-06-07 09:37:32
3. Published, but has Expired.<br>Start: 2017-05-26 09:53:20<br>Finish: 2017-05-28 09:38:03
4. Publish Item<br>Start: 2017-05-16 17:00:30

As you can see 1-3 have a description of the the current state but 4 has a description of what will happen when you toggle/click the icon
All 4 should be describing the current state and the change that will happen when you toggle/click the icon

**Comment by C-Lodder** [comment]
Funnily enough I noticed this the other day and got somewhat confused.

**Comment by brianteeman** [comment]
yes its a long stading issue but as I suspect it is quite a large change it will have to be for 4 not 3
